src/ansys/aedt/core/filtersolutions_core/optimization_goals_table.py

```diff
@@ -293,7 +293,7 @@ def remove_row(self, row_index):
 
     def set_design_goals(self):
         """Configure the optimization goal table according to the recommended goals for the current design."""
```

Comment:
Are these the default values in the table initially? If so, maybe it would be more helpful to name this `restore_design_goals`.